### PR TITLE
Documentation for Array (Parametric) Plot

### PR DESCRIPTION
**Pull request by jansilar**
The documentation build doesn't work for me, because of some memory leak of omc. Please check whether it builds correctly and the output is OK. Thanks.

### CODE REVIEW

File: UsersGuide/source/pdemodelica.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
+PDEModelia1
```
Comment (adeas31) [inline UsersGuide/source/pdemodelica.rst]: Please fix this typo.

Comment (jansilar) [reply UsersGuide/source/pdemodelica.rst]: It is called PDEModelica1 to emphasize it is just for 1-D problems and to distinguish from the extension by Saldamli which is for 3-D problems. It was not used it consistently. I amended and force-pushed a fix now.

Comment (sjoelund) [reply UsersGuide/source/pdemodelica.rst]: But it doesn't say PDEModelica1...

Comment (jansilar) [reply UsersGuide/source/pdemodelica.rst]: Sorry Martin I don't understand.

Comment (sjoelund) [reply UsersGuide/source/pdemodelica.rst]:
```diff
-PDEModelia1
+PDEModelica1
```

**Comment by jansilar** [comment]
Sure. Fixed.

**Pull request merged**